src/cpp/session/modules/SessionRCompletions.R

```diff
@@ -68,6 +68,7 @@ assign(x = ".rs.acCompletionTypes",
           COLUMN      = 27,
           R6_OBJECT   = 28, 
           DATATABLE_SPECIAL_SYMBOL = 29,
+          DATATABLE_ARGUMENT       = 30,
```

Comment:
The idea with this, instead of `ARGUMENT` is that arguments of `[.data.table` should go after the columns and the special symbols (.SD, ...) 

https://user-images.githubusercontent.com/2625526/202768547-e331254f-4e0d-4bf1-b790-b20775eae505.mov

But maybe this is not specific to `data.table` and we might want a more general way to downgrade arguments to "less important arguments" ?